src/overlays/actors/ovl_Elf_Msg2/z_elf_msg2.c

```diff
@@ -135,7 +135,7 @@ void func_8096EF98(ElfMsg2* this, PlayState* play) {
 
 void func_8096EFD0(ElfMsg2* this, PlayState* play) {
     if ((this->actor.home.rot.y < 0) && (this->actor.home.rot.y >= -0x80) &&
-        (Flags_GetSwitch(play, -this->actor.home.rot.y - 1))) {
+        Flags_GetSwitch(play, -this->actor.home.rot.y - 1)) {
```

Comment:
I wonder if `SWITCH_FLAG_NONE` could be used here in any way.

Reply:
`this->actor.home.rot.y` cannot be 0 per the condition, and a value of -0x80 "sets" SWITCH_FLAG_NONe I believe.